app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,47 @@
+class PagesController < ApplicationController
+  #before_action :set_page, except: [:index, :new, :create]  
+  before_action :set_page, only: [:show, :edit, :update, :destroy]  
+  def index
+        @pages = Page.all
+    end    
+
+    def show
+      set_page
+     
+    end    
+
+    def new
+      @page = Page.new
+    end  
+    
+    def create
+      page_params = params.require(:page).permit(:title, :body, :slug)
```

Comment:
The page_params variable is already included in the page_params method below _private_. So you won't need this variable here.